GoogleDataTransport/GDTCORLibrary/Public/GDTCOREvent.h

```diff
@@ -82,12 +85,14 @@ typedef NS_ENUM(NSInteger, GDTCOREventQoS) {
 - (nullable instancetype)initWithMappingID:(NSString *)mappingID
                                     target:(NSInteger)target NS_DESIGNATED_INITIALIZER;
 
-/** Returns the GDTCORStoredEvent equivalent of self.
+/** Write the dataObject to given URL, assign given URL to self->fileURL,
+ * and set dataObject to nil.
```

Comment:
Writes [dataObject transportBytes] to the given URL, populates fileURL with the filename, then nils the dataObject property.